api-admin/src/main/resources/api/openapi.yml

```diff
@@ -1762,6 +1747,57 @@ paths:
         '500':
           $ref: '#/components/responses/ServerError'
 
+  /api/portals/v1/{portalShortcode}/studies/{studyShortcode}/env/{envName}/dataImport/{importId}:
+    get:
+      summary: Get the import
+      tags: [ dataImport ]
+      operationId: get
+      parameters:
+        - { name: portalShortcode, in: path, required: true, schema: { type: string } }
+        - { name: studyShortcode, in: path, required: true, schema: { type: string } }
+        - { name: envName, in: path, required: true, schema: { type: string } }
+        - { name: importId, in: path, required: true, schema: { type: string, format: uuid } }
+      responses:
+        '200':
+          description: import
+          content: { application/json: { schema: { type: object } } }
+        '500':
+          $ref: '#/components/responses/ServerError'
+
+    delete:
+      summary: delete imports for a study
+      tags: [ dataImport ]
+      operationId: delete
+      parameters:
+        - { name: portalShortcode, in: path, required: true, schema: { type: string } }
+        - { name: studyShortcode, in: path, required: true, schema: { type: string } }
+        - { name: envName, in: path, required: true, schema: { type: string } }
+        - { name: importId, in: path, required: true, schema: { type: string, format: uuid } }
+      responses:
+        '200':
+          description: successful deletion
+        '500':
+          $ref: '#/components/responses/ServerError'
+
+  /api/portals/v1/{portalShortcode}/studies/{studyShortcode}/env/{envName}/dataImport/{importId}/{importItemId}:
```

Comment:
As of now not in use.
Added it incase we need to delete only a ImportItem. 
Can be removed if we think we definitely dont need it.